src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -268,7 +271,11 @@ public Command extendFully() {
   }
 
   public Command retract() {
-    return runClimber(ClimberConstants.kClimberRetractedSetpoint, 0);
+    return runOnce(
+        () -> {
+          climberServo.setAngle(ClimberConstants.kServoDisengageAngle);
+          runClimber(ClimberConstants.kClimberRetractedSetpoint, 0);
+        });
```

Comment:
You should sequence this so the servo has time to move before the the climber starts to retract. 

runClimber doesn't do anything here, because it is itself a Command that your runOnce Command is creating and throwing away (i.e. not scheduling). 

A SequentialCommand is the way to accomplish this. 
Pseudo code: 
runOnce(servo move).andThen(wait for a second or so).andThen(run climber)